notebooks/ltx-video/ltx-video.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from ov_ltx_video_helper import TextEncoderWrapper, ConvTransformerWrapper, VAEWrapper
</pre><p>could you please add comments for opportunity ti revise wrappers code without leaving notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>added</p>